docs/modules/ROOT/pages/removing.adoc

```diff
@@ -1,5 +1,15 @@
 = Removing the Desktop Synchronization Client
 
+== Removing the Configuration File
+
+When you remove the client - the configuration file remains on your system. If you then decide to install the client again, you won't have to enter the connection information.
+ 
+In case you want a clean removal of the client - delete the configuration file located at: 
+
+----
+%appdata%\owncloud\owncloud.cfg
```

Comment:
Would recommend to just remove the whole folder.

Reply:
good point

Reply:
The config dir is platform dependent. The location is documented somewhere in here.

Reply:
I think we should document it for the windows users primarily. Or for now.

Reply:
https://doc.owncloud.com/desktop/advanced_usage/configuration_file.html